lib/utils/object/clear_empties.ts

```diff
@@ -0,0 +1,16 @@
+import {GeneralObjectOrValue} from "./types";
+
+export const clearEmpties = <T>(object: GeneralObjectOrValue<T>): GeneralObjectOrValue<T> => {
+	// TODO: implement deepClone for immutability of the passed in object?
+	Object.keys(object).forEach((key) => {
```

Comment:
Do we only want this to go one level deep?

I would rely on the other methods in `utils/object` to handle this. Such as `utils.object.keys`. That way it works for nested properties.

Reply:
This method uses recursion to cycle through the keys. I tried replacing with `utils.object.keys` and it failed the nested tests.

Reply:
This method uses recursion to cycle through the keys. I tried replacing with utils.object.keys and it failed the nested tests.